modules/core/src/main/java/org/apache/ignite/internal/commandline/CommandHandler.java

```diff
@@ -769,7 +773,8 @@ private void printCacheHelp() {
 
         usageCache(LIST, "regexPattern", "[groups|seq]", "[nodeId]", op(CONFIG), op(OUTPUT_FORMAT, MULTI_LINE.text()));
         usageCache(CONTENTION, "minQueueSize", "[nodeId]", "[maxPrint]");
-        usageCache(IDLE_VERIFY, op(CMD_DUMP), op(CMD_SKIP_ZEROS), "[cache1,...,cacheN]");
+        usageCache(IDLE_VERIFY, op(CMD_DUMP), op(CMD_SKIP_ZEROS), "[cache1,...,cacheN]", op(CACHE_KIND),
+            "[ALL|SYSTEM|PERSISTENT|NOT_PERSISTENT]");
```

Comment:
You should use `op()` and `or()` in cache usage. Could you fix cache usage for LIST and CONTENTION too?

Reply:
Please use enum values in cache usage instead of strings.

Reply:
Done.